ZIGSIMPlus/Models/AppSettingModel.swift

```diff
@@ -144,11 +144,19 @@ public class AppSettingModel {
         ]
     }
 
-    public func isCameraEnabled() -> Bool {
-        return AppSettingModel.shared.isActiveByCommand[.arkit]! ||
-            AppSettingModel.shared.isActiveByCommand[.faceTracking]! ||
-            AppSettingModel.shared.isActiveByCommand[.imageTracking]! ||
-            AppSettingModel.shared.isActiveByCommand[.ndi]!
+    public func isCameraEnabled(except command: Command? = nil) -> Bool {
+        var isCameraEnabled = false
+        if command != .arkit {
+            isCameraEnabled = (isCameraEnabled || isActiveByCommand[.arkit]!)
+        }
+        if command != .ndi {
+            isCameraEnabled = (isCameraEnabled || isActiveByCommand[.ndi]!)
+        }
+        if command != .imageDetection {
+            isCameraEnabled = (isCameraEnabled || isActiveByCommand[.imageDetection]!)
+        }
+        
+        return isCameraEnabled
```

Comment:
It's a bit difficult to understand what this method is for..
How about renaming this method like `isCameraAvailable(for:)` ?

Reply:
@fand This method checks if camera(s) is set to be used.
So, how about `usesCamera(exceptBy: Command)` ?
The followings are other ideas, but I think `usesCamera` is the best:
`isCameraSet`
`accessesCamera`

Reply:
I started thinking `isCameraSet` is better considering the combination with "exceptBy"...

Reply:
Oh I skipped this line.
> How about renaming this method like `isCameraAvailable(for:)` ?

I finally got your point.
This method is also used from `CommandOutputPresenter.isCameraEnabled()` (I think you created this).
I wanted to share logic with this call.

Reply:
I've come up with a refactoring idea.
Let me see.

Reply:
I've come up with a refactoring idea.
Let me see.

Reply:
Yep, this method was created by me, but the usecase has changed and the name `isCameraEnabled` doesn't fit the case anymore 😅

Reply:
I renamed `isCameraEnabled` to `isCameraUsed` to clarify the meaning.
Also I defined `CommandAndServiceMediator.isCameraAvailable(for:)` to be more understandable.